wiki/People/ru.md

```diff
@@ -0,0 +1,6 @@
+# Люди
+
+Заглавная страница для статей о людях и группах.
+
+- [Вкладчики сообщества](Community_Contributors)
+- [Команда osu!](The_Team)
```

Comment:
should be a translation of "The Team" or "Team", not "osu! team"

it should also match the title of the russian the_team article, so either copy title from there or change it in both places